tests/unit/routes/relationship_info/test_api.py

```diff
@@ -96,6 +70,111 @@ def setup_creation_route(self, api_client, catalog_id=DEFAULT_CATALOG_ID):
             )
             assert response.status_code == HTTPStatus.CREATED
 
+    @staticmethod
+    def create_app_container(persistent, user_id, catalog_id):
```

Comment:
Would it be useful for this to be a fixture in the base test suite class?

Reply:
We could move it to base test suite class if other tests are using this. So far this is only used in `relationship_info` route.